### PR TITLE
Warnings fixed + new category

### PR DESCRIPTION
**Pull request by jan-stanek**

### CODE REVIEW

File: app/templates/macros/graphs/registrace.html [hunk 1]
```diff
@@ -11,6 +11,7 @@ <h6>Poznámky</h6>
         <li>29. 3. 2021 - otevření registrace pro pracovníky kritické infrastruktury</li>
         <li>7. 4. 2021 - otevření registrace pro pracovníky v sociálních službách</li>
         <li>14. 4. 2021 - otevření registrace pro starší 65 let</li>
+        <li>23. 4. 2021 - otevření registrace pro starší 60 let</li>
```
Comment (msusicky) [inline app/templates/macros/graphs/registrace.html:14]: Please, put here 55 from tommorow :)

File: app/templates/macros/graphs/fronta.html [hunk 2]
```diff
@@ -14,6 +14,7 @@ <h6>Poznámky</h6>
         <li>29. 3. 2021 - otevření registrace pro pracovníky kritické infrastruktury</li>
         <li>7. 4. 2021 - otevření registrace pro pracovníky v sociálních službách</li>
         <li>14. 4. 2021 - otevření registrace pro starší 65 let</li>
+        <li>23. 4. 2021 - otevření registrace pro starší 60 let</li>
```
Comment (msusicky) [inline app/templates/macros/graphs/fronta.html:17]: Put here 55 from tommorow please.

**Review by msusicky — CHANGES_REQUESTED** [review]

File: app/templates/macros/graphs/fronta.html (hunk 2 above)
Comment (jan-stanek) [reply app/templates/macros/graphs/fronta.html:17]: done

File: app/templates/macros/graphs/registrace.html (hunk 1 above)
Comment (jan-stanek) [reply app/templates/macros/graphs/registrace.html:14]: done

**Review by msusicky — APPROVED** [review]

**Pull request merged**